hack/verify-examples-kind.sh

```diff
@@ -51,11 +54,19 @@ res=0
 KIND_CREATE_ATTEMPTED=true
 kind create cluster --name "${CLUSTER_NAME}" || res=$?
 
+cat <<EOF >config/webhook/kustomization.yaml
+resources:
+  - 0-namespace.yaml
+  - certificate_config.yaml
+  - admission_webhook.yaml
+images:
+  - name: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.0
```

Comment:
```suggestion
  - name: gcr.io/k8s-staging-gateway-api/admission-server:v0.5.1
```

Reply:
Would be even better if this was a var that could be set at the top of the file.

Reply:
Yes, it makes sense. I'll think about it.